pages/tutorials/yaml/species.rst

```diff
@@ -0,0 +1,209 @@
+.. slug: yaml-species
+.. title: Elements and Species
+.. has_math: true
+
+.. jumbotron::
+
+   .. raw:: html
+
+      <h1 class="display-3">Elements and Species</h1>
+
+   .. class:: lead
+
+      A description of how elements and species are defined in YAML input files
+
+.. _sec-yaml-guide-elements:
+
+Elements
+========
+
+Cantera provides built-in definitions for the chemical elements, including
+values for their atomic weights taken from IUPAC / CIAAW. These elements can be
+used by speciying the corresponding atomic symbols when specifying the
+composition of species.
+
+In order to give a name to a particular isotope or a virtual element
+representing a surface site, a custom ``element`` entry can be used. The default
+location for ``element`` entries is the ``elements`` section of the input file.
+Elements defined in this section will automatically be considered for addition
+to phases defined in the same file. Elements can be defined in other sections of
+the input file if those sections are named explicitly in the ``elements`` field
+of the phase definition.
+
+An element entry has the following fields:
+
+- ``symbol``: The symbol to be used for the element, e.g. when specifying the
+  composition of a species.
+- ``atomic-weight``: The atomic weight of the element, in unified atomic mass
+  units (dalton)
+- ``atomic-number``: The atomic number of the element. Optional.
+- ``entropy298``: The standard molar entropy of the element at 298.15 K. Optional.
+
+An example ``elements`` section:
+
+.. code:: yaml
+
+    elements:
+    - symbol: C13
+      atomic-weight: 13.003354826
+      atomic-number: 12
+    - symbol: O-18
+      atomic-weight: 17.9991603
+
+.. _sec-yaml-guide-species:
+
+Species
+=======
+
+A species entry in Cantera is used to specify the name, composition,
+thermodynamic, and transport properties of an individual species.
```

Comment:
Should Cantera/enhancements#14 be considered for 2.5.0?

Reply:
Sure, I'd consider including an implementation if someone made a PR, but I don't think it's a blocker for the release.